### ISSUE TITLE
ComboBox DAP Violations: 1. The WAI-ARIA widget with img role must have a nonempty alternate text or a label 2. The WAI-ARIA role(s) and/or attribute(s) identifier are not valid for the element identifier

### ISSUE THREAD

**Issue by katindoy** [carbon-design-system/carbon#6080]
## Environment

> Operating system: Mac OS 10.15.3

> Browser: Chrome 81.0.4044.122

## Detailed description

> What version of the Carbon Design System are you using?
carbon-components-react v7.11.3

> What did you expect to happen?
DAP violations from ComboBox component:
1. The WAI-ARIA widget with img role must have a nonempty alternate text or a label 
2. The WAI-ARIA role(s) and/or attribute(s) identifier are not valid for the element identifier

## Steps to reproduce the issue
1. Go to sandbox at https://codesandbox.io/s/admiring-cohen-s0zyz
2. Run DAP and notice the following violations:
     - input field: The WAI-ARIA role(s) and/or attribute(s) combobox, aria-autocomplete, aria-expanded, aria-activedescendant are not valid for the element input. See rule help for guidance.
     - svg: The WAI-ARIA widget with img role must have a nonempty alternate text or a label.

![Screen Shot 2020-05-13 at 4 49 45 PM](https://user-images.githubusercontent.com/25806250/81877426-e5d77a80-9539-11ea-8283-cf71220bd720.png)

**Comment by joshblack** [comment]
cc @dakahn was this closed by your downshift updates?

**Comment by dakahn** [comment]
Covered in our updates! :D
